RevertConditions/SafeMath/README.md

```diff
@@ -0,0 +1,17 @@
+# SafeMath Reverts
+SafeMath is a Solidity library developed by OpenZeppelin that wraps over Solidity math operation to prevent undefined behaviors.
+Instead, when such a state is reached (for example division by 0) the SafeMath will be responsible to revert the transaction.
+From Solidity 8.0 onward, SafeMath is integrated be default into the language and is not needed to be imported explicitly to use it.
```

Comment:
1. Change "SafeMath is integrated ~~be~~ **by** default"
2. I'm not sure saying "SafeMath is integrated by default into the language" is precise

Reply:
I changed the phrasing a bit.